### PR TITLE
fix: broken contents of "Full Node" section

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

File: content/docs/full-node/manage-a-validator/_index.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 
 Validators and their delegators earn the following rewards:
 
-- [Gas](../../learn/fees.md#gas): Fees added on to each transaction to avoid spamming and pay for computing power. Validators set minimum gas prices and reject transactions that have implied gas prices below this threshold.
+- [Gas]({{< ref "/docs/learn/fees#gas" >}}): Fees added on to each transaction to avoid spamming and pay for computing power. Validators set minimum gas prices and reject transactions that have implied gas prices below this threshold.
 
-- [Inflation rewards](../../develop/module-specifications/spec-mint.md): Every block, new XPLA is minted and released to validators and delegators as staking rewards. The rate for the minting of this new XPLA is fixed at 7% per year. 
+- [FIXME!! Inflation rewards](../../develop/module-specifications/spec-mint.md): Every block, new XPLA is minted and released to validators and delegators as staking rewards. The rate for the minting of this new XPLA is fixed at 7% per year.
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/_index.md]: the link should be fixed to `../../develop/core-modules/mint`

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
 weight: 20
-title: FAQ
+title: Validator FAQ
 ---
 
 # Validator FAQ
 
-::: {important}
+{{< hint warning >}}
+**Inportant**
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md]: typo

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
-## General concepts
+## General Concepts
 
-### What is a validator?
+### What Is a Validator?
 
 The Xpla Core is powered by the Tendermint consensus. Validators run full nodes, participate in consensus by broadcasting votes, commit new blocks to the blockchain, and participate in governance of the blockchain. Validators are able to cast votes on behalf of their delegators. A validator's voting power is weighted according to their total stake.
 
-### What is a full node?
+### What is a Full Node?
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md]: here and the rest be verbs in this page's titles need to be started with a capital letter.

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 4]
```diff
@@ -129,15 +131,15 @@ Ask your validator for their xplavaloper address.
 
 Be careful when delegating to validators outside of the active set. Some inactive validators may be jailed or are no longer supported. Station only displays active or recently active validators who are able to participate in consensus.
 
-### Is there a faucet?
+### Is There a Faucet?
 
-Use the [Xpla faucet](https://faucet.c2x.world/) to obtain coins for the testnet.
+Use the [Xpla faucet](https://faucet.c2x.io/) to obtain coins for the testnet.
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md]: I guess the domain name is `xpla.io`?

**Comment by caramis** [comment]
Good review 👍 @jhlee-young 
All of the above reviews are fixed.

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**